src/codeu/chat/server/Controller.java

```diff
@@ -34,6 +34,11 @@
 
 public final class Controller implements RawController, BasicController {
 
+  public static final byte USER_TYPE_CREATOR = 0x07;
```

Comment:
Should this be `0x04`? You only need a single bit to represent each role, and it's possible for a user to be a creator without being an owner or a user, right? If you represent the creator, owner, and user roles as `0b100`, `0b010`, and `0b001` respectively, your code can combine those in any possible combination.